### PR TITLE
fix: Pin module version used for addon

### PR DESCRIPTION
**Pull request by bryantbiggs**
Ref https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/1630

### CODE REVIEW

**Comment by bryantbiggs** [comment]
@manuelbcd can we get this quickly reviewed and merged, please

**Comment by fcarta29** [comment]
I think the following needs to be updated as well 
https://github.com/sysdiglabs/terraform-eksblueprints-sysdig-addon/blob/fe3dc38639eefd35c139ec0c74c5539aad2acc18/blueprints/monitor-alerts/README.md?plain=1#L105

**Comment by manuelbcd** [comment]
Hey @bryantbiggs @fcarta29 , let me test it asap today and give feedback, thanks!

File: blueprints/lab-security/main.tf [hunk 1]
```diff
@@ -126,7 +126,7 @@ module "eks_blueprints" {
 }
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints?ref=v4.21.0//modules/kubernetes-addons"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints/modules/kubernetes-addons?ref=v4.32.0"
```
Comment (bryantbiggs) [inline blueprints/lab-security/main.tf]:
```suggestion
  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.32.0"
```

File: blueprints/monitor-alerts/main.tf [hunk 2]
```diff
@@ -132,7 +132,7 @@ module "eks_blueprints" {
 }
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints?ref=v4.21.0//modules/kubernetes-addons"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints/modules/kubernetes-addons?ref=v4.32.0"
```
Comment (bryantbiggs) [inline blueprints/monitor-alerts/main.tf]:
```suggestion
  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.32.0"
```

**Comment by manuelbcd** [comment]
@bryantbiggs @fcarta29 this version shed several errors when testing with Terraform plan, any suggestion?

```
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 92, in module "eks_blueprints":
│   92:   cluster_name    = local.cluster_name
│ 
│ An argument named "cluster_name" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 93, in module "eks_blueprints":
│   93:   cluster_version = "1.23"
│ 
│ An argument named "cluster_version" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 95, in module "eks_blueprints":
│   95:   vpc_id             = module.vpc.vpc_id
│ 
│ An argument named "vpc_id" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 96, in module "eks_blueprints":
│   96:   private_subnet_ids = module.vpc.private_subnets
│ 
│ An argument named "private_subnet_ids" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 98, in module "eks_blueprints":
│   98:   managed_node_groups = {
│ 
│ An argument named "managed_node_groups" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 125, in module "eks_blueprints":
│  125:   tags = local.tags
│ 
│ An argument named "tags" is not expected here.
```

**Comment by fcarta29** [comment]
@manuelbcd - Were you able to resolve the issues you were seeing? If so any chance we can get this PR merged and a release cut with the pinned version?

**Comment by manuelbcd** [comment]
sure Frank, let me have a look to it

**Comment by manuelbcd** [comment]
Changes applied one by one in a different branch. Thank you

**Pull request closed without merging**